silkworm/silkrpc/ethdb/cbor.cpp

```diff
@@ -15,6 +15,8 @@
 */
 
 #include "cbor.hpp"
+#include <cbor/cbor.h>
+#include "listener_cbor_log.h"
```

Comment:
Rename `listener_cbor_log.h` as `log_cbor_listener.hpp`

Reply:
moved the new class to cbor.cpp